src/index/updater.rs

```diff
@@ -34,6 +34,7 @@ pub(crate) struct Updater {
   height: u64,
   index_sats: bool,
   sat_ranges_since_flush: u64,
+  options: Options,
```

Comment:
Do we really need to make `options`  a member of the `Updater` struct? It's only used one time on line 101, `let rx = Self::fetch_blocks_from(index, self.height, self.index_sats, self.options.clone())?;` where we are already passing in `index` which has the `options` as a member already.

Reply:
good point, I'll try to remove it.